dclab/features/inert_ratio.py

```diff
@@ -1,23 +1,26 @@
 """Computation of inertia ratio from contour data"""
+from __future__ import annotations
+
 import numpy as np
+import numpy.typing as npt
 import scipy.spatial as ssp
 
 
-def cont_moments_cv(cont,
-                    flt_epsilon=1.19209e-07,
-                    dbl_epsilon=2.2204460492503131e-16):
+def cont_moments_cv(cont: npt.NDArray,
+                    flt_epsilon: float = 1.19209e-07,
+                    dbl_epsilon: float = 2.2204460492503131e-16) -> dict:
     """Compute the moments of a contour
 
     The moments are computed in the same way as they are computed
     in OpenCV's `contourMoments` in `moments.cpp`.
 
     Parameters
     ----------
-    cont: array of shape (N,2)
+    cont: shape (N,2)
```

Comment:
keep the `array of shape (N, 2)`, only putting `shape (N, 2)` might be confusing. Or put the shape in the docstring.